### PR TITLE
Make description formatting more consistent and remove unnecessary styling

### PR DESCRIPTION
**Pull request by Cuingamehtar**
These will probably be mostly invisible to the end user.

### CODE REVIEW

File: packs/howl-of-the-wild-bestiary/alicorn.json [hunk 1]
```diff
@@ -206,7 +206,7 @@
                 "damage": {},
                 "defense": null,
                 "description": {
-                    "value": "<p>You can communicate telepathically with creatures within 30 feet. Once you establish a connection by communicating with a creature, the communication is two-way. You can communicate only with creatures that share a language with you.</p>\n<hr />\n<p><strong>Heightened (6th)</strong> You can communicate telepathically with creatures using shared mental imagery even if you don't share a language <em>telepathy</em> loses the linguistic trait.</p>"
+                    "value": "<p>You can communicate telepathically with creatures within 30 feet. Once you establish a connection by communicating with a creature, the communication is two-way. You can communicate only with creatures that share a language with you.</p>\n<hr />\n<p><strong>Heightened (6th)</strong> You can communicate telepathically with creatures using shared mental imagery even if you don't share a language; <em>telepathy</em> loses the linguistic trait.</p>"
```
Comment (simonward) [inline packs/howl-of-the-wild-bestiary/alicorn.json:209]: Telepathy should have the semicolon to match the source

Comment (Cuingamehtar) [reply packs/howl-of-the-wild-bestiary/alicorn.json:209]: Yes.
I check with both the source book and AoN before committing.

Comment (simonward) [reply packs/howl-of-the-wild-bestiary/alicorn.json:209]: My mistake - you’re adding the semicolon, not removing it.

**Comment by Cuingamehtar** [comment]
I think I'll stop for now. There is always _one more thing_, but that can wait for some other time.

**Pull request closed without merging**